src/include/libpmem2.h

```diff
@@ -276,6 +277,13 @@ void pmem2_badblock_context_delete(
 int pmem2_badblock_clear(struct pmem2_badblock_context *bbctx,
 		const struct pmem2_badblock *bb);
 
+#ifdef PMEM2_USE_MINIASYNC
```

Comment:
maybe we could simply have a define inside miniasync like `MINIASYNC_PRESENT`, and then simply instruct people to include miniasync (before libpmem2.h) before pmem2.

Reply:
It will be a more "leaky abstraction", than my solution